[bug] Error occurred when configuring `app.windows.windowEffects.color`

### Describe the bug

When `app.windows.windowEffects.color` is configured, tauri cannot compile.

### Reproduction

1. Add the following configuration to the `app.windows` section of `tauri.conf.json`

```json
"windowEffects": {
  "effects": ["acrylic", "blur", "hudWindow"],
  "state": "active",
  "color": [0, 0, 0, 50]
}
```

2. Run tauri.

### Expected behavior

_No response_

### Full `tauri info` output

```text
[✔] Environment
    - OS: Windows 10.0.22631 X64
    ✔ WebView2: 122.0.2365.59
    ✔ MSVC: Visual Studio Community 2022
    ✔ rustc: 1.76.0 (07dca489a 2024-02-04)
    ✔ cargo: 1.76.0 (c84b36747 2024-01-18)
    ✔ rustup: 1.26.0 (5af9b9484 2023-04-05)
    ✔ Rust toolchain: stable-x86_64-pc-windows-msvc (default)
    - node: 20.10.0
    - pnpm: 8.15.4
    - yarn: 3.6.1
    - npm: 10.2.3

[-] Packages
    - tauri [RUST]: 2.0.0-beta.8
    - tauri-build [RUST]: 2.0.0-beta.6
    - wry [RUST]: 0.37.0
    - tao [RUST]: 0.26.1
    - @tauri-apps/api [NPM]: 2.0.0-beta.3
    - @tauri-apps/cli [NPM]: 2.0.0-beta.6

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:1420/
    - framework: Vue.js
    - bundler: Vite
```


### Stack trace

```text
error[E0425]: cannot find function, tuple struct or tuple variant `Color` in crate `tauri::utils`
  --> src\lib.rs:44:14
   |
44 |         .run(tauri::generate_context!())
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in `tauri::utils`
   |
   = note: this error originates in the macro `tauri::generate_context` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider importing this tuple struct
   |
1  + use tauri::window::Color;
   |

For more information about this error, try `rustc --explain E0425`.
```


### Additional context

_No response_

--- a/.changes/color-context-generation.md
+++ b/.changes/color-context-generation.md
@@ -1,0 +1,6 @@
+---
+'tauri-utils': 'patch:bug'
+'tauri': 'patch:bug'
+---
+
+Fix compile time error in context generation when using `app.windows.windowEffects.color`

--- a/core/tauri-utils/src/config.rs
+++ b/core/tauri-utils/src/config.rs
@@ -2096,7 +2096,7 @@ mod build {
   impl ToTokens for Color {
     fn to_tokens(&self, tokens: &mut TokenStream) {
       let Color(r, g, b, a) = self;
-      tokens.append_all(quote! {::tauri::utils::Color(#r,#g,#b,#a)});
+      tokens.append_all(quote! {::tauri::utils::config::Color(#r,#g,#b,#a)});
     }
   }
   impl ToTokens for WindowEffectsConfig {

